docs/index.md

```diff
@@ -651,21 +651,31 @@ The process begins with a QR code or `openid://` link. The only differences are:
     ```
 ---
 
-## Potential Extensions
+### Paper-based fallback: Health Card Links
+* Allows basic use of health cards for users without a smartphone
+* Allow smartphone-enabled users to print a usable backup
+* Allows full health card contents to be shared with a verifier
+* Requires that the verifier is online
+* Does not capture a digital record of a signed request (no DID-SIOP Request flow)
+* Does not capture a digital record of the presentation  (no Verifiable Presentation flow)
 
-### Fallback for smartphone-based offline presentation
+In this flow, the user obtains a physical or virtual *Health Card Link* QR code. This code can be shared with verifying parties or loaded into a new wallet app by scanning the QR code. The *Health Card Link* is a QR code whose payload consists of a minified JSON payload with three properties:
 
-We should be able to specify additional "return paths" in the DID SIOP workflow that don't depend on an HTTP upload but instead rely on local transfer (e.g., via NFC or bluetooth)
+1. `type` value of `https://smarthealth.cards#1`, identifying this QR code as a *Health Card Link*
+1. `url` value that dereferences to an encrypted Health Card (following the same JWE-based encryption scheme described above)
+1. `jwk` value conveying a private key (`"alg": "ECDH-ES", "crv": "P-256"`) capable of decrypting the encrypted Health Card
 
-### Fallback for users without a smartphone
+Consumers can obtain a Health Card Link through two main workflows:
+* **For consumers *without* a smartphone**, issuers or other trusted "Health Card Link Creators" can create a QR code and print it. In this process, the Health Card Creator generates or obtains a Health Card; generates an encryption key; encrypts the Health Card; and causes the encrypted Health Card to be hosted at some URL. The Health Card Creator then populates a Health Card Link with the three required proeprties, printing the resulting QR code or otherwise sharing it with the consumer.
```

Comment:
```suggestion
* **For consumers *without* a smartphone**, issuers or other trusted "Health Card Link Creators" can create a QR code and print it. In this process, the Health Card Creator generates or obtains a Health Card; generates an encryption key; encrypts the Health Card; and hosts the encrypted Health Card at some URL. The Health Card Creator then populates a Health Card Link with the three required properties, printing the resulting QR code or otherwise sharing it with the consumer.
```

Would it be less confusing to use the term _Issuer_ rather than _Creator_ here? i.e. "Health Card Link Issuer"

Reply:
Maybe; I want to avoid using "issuer" in two contexts though.